src/Tools/BuildValidator/Util.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+
+namespace BuildValidator
+{
+    internal sealed record PortableExecutableInfo(string FilePath, Guid Mvid, bool IsReadyToRun);
```

Comment:
Consider renaming the file to 'PortableExecutableInfo.cs', moving 'Util.GetPortableExecutableInfo' to a 'PortableExecutableInfo.Create' method, then eliminating the 'Util' class.

Reply:
I think that having a type per file with records is going to de-value them a bit. A record is meant to be a simple concise declaration. Having them be per file means that we pretty much lose that as we have to include the copyright, namespace, using, etc ... Basically bringing more and more boiler plate back into the feature that was meant to avoid it. 

Prefer we think about an approach where we can keep the simplicity here. Possibly just having a file for all the simple records as long as they remain ... simple. Push them out to separate files once they hit some levele of complexity.